### DISCUSSION TITLE
HW 5 part 2.1

### DISCUSSION THREAD

**Discussion by UniquelyAbbi** [cooperrc/computational-mechanics#67] [General]
Hello, im working on the homework  but i am now stuck. I am trying to use the book but i only confused myself more.

![image](https://user-images.githubusercontent.com/106557922/176775027-6c75d65c-c3fc-4011-b4e0-057a158a6948.png)

This is what I have

**Comment by UniquelyAbbi** [comment]
There are some terms in this that do not ring a bell which is probably why i am struggling. Thoughts and opinnions?

**Reply by feliciones** [reply]
This looks good to me, only thing you will have to do is set delta x at the different distance values, 10,5, and 1. You can do a for loop for this or because it was only 3 different instances I just hardcoded it.

I'm also working on this problem right now so maybe we can help each other through it.

**Reply by UniquelyAbbi** [reply]
I would like that, thank you.

**Reply by feliciones** [reply]
This is what I have so far @UniquelyAbbi:

![image](https://user-images.githubusercontent.com/106564968/176966194-0928fa2d-1389-49c1-b736-76df93f4f5e5.png)

![image](https://user-images.githubusercontent.com/106564968/176966235-19f818d3-a706-4e90-ab71-3a21453bcd64.png)

**Comment by cooperrc** [comment]
I don't see any errors (Python or Heat transfer). 

Are you stuck on the [finite difference](https://cooperrc.github.io/computational-mechanics/module_05/01_Revisiting_derivatives.html#central-difference) approach or the [heat transfer](https://cooperrc.github.io/computational-mechanics/module_05/02_Keep_it_steady.html#one-dimensional-steady-state-differential-equations) application?
